lib/views/BookmarkListView.dart

```diff
@@ -13,21 +14,31 @@ class BookmarkListView
     extends AuthenticatedView<HomeWidget, BookmarkController> {
   const BookmarkListView(state, {Key key}) : super(state, key: key);
 
+  static int subtitleLength = 42;
+  static int titleLength = 70;
```

Comment:
Could you make those `const`?
```suggestion
  static const int subtitleLength = 42;
  static const int titleLength = 70;
```